ocs_ci/templates/mcg/java/s3test/src/main/java/amazons3/s3test/ChunkedUploadApplication.java

```diff
@@ -0,0 +1,231 @@
+package amazons3.s3test;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.SDKGlobalConfiguration;
+import com.amazonaws.SdkClientException;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.CompleteMultipartUploadRequest;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.InitiateMultipartUploadRequest;
+import com.amazonaws.services.s3.model.InitiateMultipartUploadResult;
+import com.amazonaws.services.s3.model.PartETag;
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import com.amazonaws.services.s3.model.UploadPartRequest;
+import com.amazonaws.services.s3.model.UploadPartResult;
+
+import software.amazon.awssdk.utils.Pair;
+
+/**
+ * Hello world!
+ *
+ */
+//public class App
+//{
+//    public static void main( String[] args )
+//    {
+//        System.out.println( "Hello World!" );
+//    }
+//}
+
+public class ChunkedUploadApplication {
+
+	public static void main(String[] args) throws NoSuchAlgorithmException, IOException {
+		System.setProperty(SDKGlobalConfiguration.DISABLE_CERT_CHECKING_SYSTEM_PROPERTY, "true");
+
+		System.out.println("Endpoint: "+ args[0]);
+		System.out.println("Access Key: "+ args[1]);
+		System.out.println("Secret Key: "+ args[2]);
+		System.out.println("bucket name: "+ args[3]);
+		System.out.println("Is multipart?: "+ args[4]);
+
+		String endpoint = args[0];
+		String ACCESS_KEY = args[1];
+		String SECRET_KEY = args[2];
+		String bucketName = args[3];
+		Boolean isMultipart = false;
+
+		if (args[4] == "True")
+			isMultipart = true;
+		AmazonS3 s3Client = configureS3Client(endpoint, ACCESS_KEY, SECRET_KEY);
+		Boolean test1 = testCycle(s3Client, bucketName, isMultipart); // regular bucket + regular upload
```

Comment:
please change the boolean name of test1 to be something more meaningful like "testResult" or "testPassed"